### ISSUE TITLE
Suggestion: Code styling

### ISSUE THREAD

**Issue by hesselberg** [Novicell/novicell-frontend#298]
Not sure where else to put this, so here goes.

It seems we don't have a set code style in place, and as of that, eslint, stylelint and Prettier (if installed) are all fighting to control what the formatting and code style should be.

I suggest we configure to use Airbnbs code style, and make it a requirement to install Prettier. 

Thoughts?

**Comment by jhavmoeller** [comment]
I think there is a code style in place right now?

- eslint for javascript extending the airbnb and eslint recommended
- stylelint for styles. 

Doesnt it work?

**Comment by hesselberg** [comment]
@jhavmoeller you're totally right, honestly I didn't check if it was there or not. My bad. That being said, I still have the issue of Prettier fighting over eslint/stylelint sometimes, not to mention Stylelint shouting and screaming at me.

Do we need stylelint, if everyone goes with Prettier and a default config for Prettier? I'm not sure which way is the best way to go

**Comment by Dan9boi** [comment]
I think we should look at for example how NUXT is using prettier to use the ESlint configs, and then just use prettier (remove stylelint). That way we have ESlint for JS and Prettier for (mostly) css.
Anyone disagrees with this?
